applications/callflow/src/cf_endpoint.erl

```diff
@@ -226,6 +226,17 @@ merge_attributes([<<"caller_id">> = Key|Keys], Account, Endpoint, Owner) ->
             CallerId = wh_json:set_value([<<"emergency">>, <<"number">>], Number, Merged),
             merge_attributes(Keys, Account, wh_json:set_value(Key, CallerId, Endpoint), Owner)
     end;
+merge_attributes([<<"do_not_disturb">> = Key|Keys], Account, Endpoint, Owner) ->
+    AccountAttr = wh_json:is_true([Key, <<"enabled">>], Account, 'false'),
+    EndpointAttr = wh_json:is_true([Key, <<"enabled">>], Endpoint, 'false'),
+    OwnerAttr = wh_json:is_true([Key, <<"enabled">>], Owner, 'false'),
+    Dnd = case {AccountAttr, OwnerAttr, EndpointAttr} of        
```

Comment:
Why not `Dnd = AccountAttr orelse OwnerAttr orelse EndpointAttr`? Or in the reverse order if EndpointAttr should take priority.